Assets/Scripts/UI/Sidebar/TextPopout.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using Netherlands3D.Coordinates;
+using TMPro;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    public class TextPopout : MonoBehaviour
```

Comment:
I expanded on the initial popout as introduced by Protofunk to include a script for positioning it; and even to have it stick to a set of world coordinates